src/components/Link/Link.tsx

```diff
@@ -0,0 +1,26 @@
+import { FC } from "react";
+import { useConfig } from "../../hooks";
+
+interface LinkProps {
+  href: string;
+  children: React.ReactNode;
+}
+
+export const Link: FC<LinkProps> = ({ href, children }) => {
+  const config = useConfig();
+
+  return (
+    <a
```

Comment:
Do you mean to return from this Link component a Link component from RadixUI library? If you meant to use Link from Radix directly in components, it will require config import for places which don't really need to have config imported.

Reply:
The only problem with this solution is that we cannot pass an exact color from a config to CSS classname

Reply:
Changes are available for re-review!